### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by FelixWeis**

### CODE REVIEW

File: src/inscription.rs [hunk 1]
```diff
@@ -490,7 +491,7 @@ mod tests {
       .into_script();
 
     assert_eq!(
-      InscriptionParser::parse(&Witness::from_vec(vec![script.into_bytes(), Vec::new()])),
+      InscriptionParser::parse(&Witness::from_slice(&vec![script.into_bytes(), Vec::new()])),
```
Comment (FelixWeis) [inline src/inscription.rs:518]: this test is probably the best to check if inscriptions are recognized if multiple checksigs are done before the envelope

**Comment by FelixWeis** [comment]
can you check why the 3 tests fail? seems rust-bitcoin v0.30.0 related @fjahr

File: src/inscription.rs (hunk 1 above)
Comment (fjahr) [reply src/inscription.rs:518]: By recognized you just mean it's a valid inscription and doesn't error, right? I.e. it doesn't do anything special. Then yeah, looks good.

**Comment by fjahr** [comment]
> can you check why the 3 tests fail? seems rust-bitcoin v0.30.0 related @fjahr

Fixed here: https://github.com/ordilabs/ord-mini/commit/f3568a7cf829a5f2ab5a9f8473ce082bdca432f9

**Review by fjahr — APPROVED** [review]
ACK

**Pull request merged**